WriteHeader/calibration_tool.py

```diff
@@ -0,0 +1,105 @@
+from PIL import Image as image
+from PIL.TiffTags import TAGS
+import numpy
+import os
+import SimpleITK as sitk
+
+class CalibrationTool:
+    """Read and store calibration data from camera calibration"""
+
+    def __init__(self):
+        self.file = []
+
+        self.principle_dist = 0
+        self.x_offset = 0
+        self.y_offset = 0
+        self.pixel_size = 0
+
+        self.xray_position = []
+        self.image_normal = []
+        self.image_up = []
+
+        self.camera_path = 0
+        self.image_all = []
+
+        self.direction_matrix = []
+
+    def read_file(self, base_dir, file_dir, filename):
+        """Given filename, read all lines in file"""
+
+        path = base_dir + file_dir + filename
+        open_file = open(path, "r")
+        self.file = open_file.readlines()
+
+    def parse_file(self):
+        """Parse and clean all values from the calibration file"""
+
+        self.principle_dist = self.file[1].strip()
+        self.x_offset = float(self.file[2].strip())
+        self.y_offset = float(self.file[3].strip())
+        self.pixel_size = float(self.file[4].strip())
+
+        self.xray_position = self.file[6].strip().split()
+        self.image_normal = self.file[7].strip().split()
+        self.image_up = self.file[8].strip().split()
+
+    def get_calibration_data(self, base_dir, file_dir, filename):
+        self.read_file(base_dir, file_dir, filename)
+        self.parse_file()
+        self.get_direction()
+
```

Comment:
Based on my little experience with itk coding style, you are not getting_calibration_data but you are setting your calibration information. 
GetCalibrationData() should be used when you want to retrieve the information about the calibration. The naming of this definition is a bit confusing. I suggest set_calibration_data.